app/src/test/java/com/fcaputo/parkingapp/presenterTests/ReservationPresenterTest.kt

```diff
@@ -0,0 +1,165 @@
+package com.fcaputo.parkingapp.presenterTests
```

Comment:
a package should never be in camel case. It should be just presenter as you are in the tests